CNAME

```diff
@@ -1 +0,0 @@
-kottans.org
```

Comment:
hmmm, not sure why are you deleting this?
it is a part of the domain policy at github to use a custom domain instead of .github.io

Reply:
https://medium.com/@hossainkhan/using-custom-domain-for-github-pages-86b303d3918a - as an example of how to config it

Reply:
Done. 
Thank you!